### PR TITLE
Set host and port as required

### PR DESCRIPTION
**Pull request by scalarindetail**
The constructor will throw an exception if the host or the port are missing.

### CODE REVIEW

File: scalardl-web-client-sdk.js [hunk 1]
```diff
@@ -89,8 +91,8 @@ class ClientService {
    * @return {Promise<ClientServiceResponse>}
    */
   async registerCertificate() {
-    const request = new CertificateRegistrationRequestBuilder()
-        .withCertHolderId(this.certHolderId)
+    const request = new CertificateRegistrationRequestBuilder().withCertHolderId(
```
Comment (feeblefakie) [inline scalardl-web-client-sdk.js]: Is this change from lint ?

File: scalardl-web-client-sdk.js [hunk 2]
```diff
@@ -32,9 +32,11 @@ class ClientService {
     /** @const */
     this.properties = properties;
     /** @const */
-    this.serverHost = properties['scalar.ledger.client.server_host'];
+    this.serverHost = this.getRequiredProperty_(properties,
```
Comment (feeblefakie) [inline scalardl-web-client-sdk.js]: Is this `_` a style for internal method or something ?

Comment (scalarindetail) [reply scalardl-web-client-sdk.js]: Exactly, private methods should end with a '_' character. 
cf :  https://google.github.io/styleguide/jsguide.html#naming-method-names

File: scalardl-web-client-sdk.js (hunk 1 above)
Comment (scalarindetail) [reply scalardl-web-client-sdk.js]: It was but I reverted it as it was better before.

**Pull request merged**